pubnub-kotlin/pubnub-kotlin-core-api/src/commonMain/kotlin/com/pubnub/api/models/consumer/pubsub/files/PNFileEventResult.kt

```diff
@@ -45,6 +45,9 @@ class PNFileEventResult(
         if (file != other.file) {
             return false
         }
+        if (customMessageType != other.customMessageType) {
```

Comment:
if you modify equals() you MUST modify hashCode() as well to match

Reply:
Good catch. I will fix that.